### DISCUSSION TITLE
Requerimentos 1ra entrega

### DISCUSSION THREAD

**Discussion by SantyPerezzz** [dds-jv/foro#419] [TPA - 2024] [answered]
Buenas, en cuanto a los requerimientos de la primera entrega, debemos implementar una interfaz por consola para acceder a las funcionalidades del programa, o solo debemos implementar dichas funcionalidades en las clases para luego acceder a ellas en una futura entrega donde implementemos una interfaz para el usuario?

**Comment by flbulgarelli** [comment, marked as answer]
:wave: ¡Hola!

No hay que implementar ningún tipo de UI, sólo el modelo de dominio. 

¡Saludos!
